src/chat/chatbox.html

```diff
@@ -37,6 +37,7 @@
 	<!-- <script src="/scripts/bttvIntegration.mjs" type="module"></script> -->
 	<script src="/scripts/color.mjs" type="module"></script>
 	<script src="/scripts/handleChat.mjs" type="module"></script>
+	<script src="/scripts/pronouns.mjs" type="module"></script>
```

Comment:
Should scripts in this block that are unused unless a specific query parameter is set be loaded on every request?

Same treatment as the `demo` script, maybe?
`{% if eleventy.serverless.query.DEMO %}`